### DISCUSSION TITLE
Monocle or float – which one is better to peek a window?

### DISCUSSION THREAD

**Discussion by piotr-dobrogost** [anametologin/krohnkite#88] [General]
I somehow thought that when I want to _peek_ a window for a while (have it maximum size) I should float it. But maybe in such a situation it's better to change layout to Monocle with this window being active and then switch back to the original layout?

**Comment by anametologin** [comment]
1. A lot of utility windows have the float window state , so I cannot touch them and change its behavior. 
2. Monocle Layout is toggled layout. I use it if I want enlarge a one window and come back to previous layout after this.

Why are you don't want set `maximize` a window?

**Reply by piotr-dobrogost** [reply]
> Monocle Layout is toggled layout. I use it if I want enlarge a one window and come back to previous layout after this.

I get it but I would like to avoid switching layout just to _enlarge one window_ because then I have to remember to switch back to the original layout.

> Why are you don't want set `maximize` a window?

Because as I wrote in issue #151

> I find having to unmaximize a window after having maximized it temporarily to do something quickly cumbersome.

Now, I think I found the solution I was looking for when asking this question. The solution is to minimize all windows but the active (focused) one. This allows to eat the cake (maximize the window) and have it, too (stay in the current layout).
